applications/nrf5340_audio/doc/fota.rst

```diff
@@ -7,66 +7,38 @@ Configuring and testing FOTA upgrades for nRF5340 Audio applications
    :local:
    :depth: 2
 
-All nRF5340 Audio applications share the same configuration and testing procedures for FOTA upgrades.
+The nRF5340 Audio applications all support FOTA upgrades, and the application implementation is based on the procedure described in :ref:`ug_nrf53_developing_ble_fota`.
 
 Requirements for FOTA
 *********************
 
-To test Firmware Over-The-Air (FOTA), you need an Android or iOS device with the `nRF Connect Device Manager`_ app installed.
-
-To enable the external flash DFU and do FOTA upgrades for the application core and the network core at the same time, you need an external flash shield.
+If the application is running on the nRF5340 Audio DK an external flash shield is needed in order to update both the application and network core at the same time.
 See `Requirements for external flash memory DFU`_ in the nRF5340 Audio DK Hardware documentation for more information.
 
 .. _nrf53_audio_app_configuration_configure_fota:
 
 Configuring FOTA upgrades
 *************************
 
-.. caution::
-	Firmware based on the |NCS| versions earlier than v2.1.0 does not support DFU.
-	FOTA is not available for those versions.
-
-	You can test performing separate application and network core upgrades, but for production, both cores must be updated at the same time.
-	When updates take place in the inter-core communication module (HCI IPC), communication between the cores will break if they are not updated together.
-
-You can configure Firmware Over-The-Air (FOTA) upgrades to replace the applications on both the application core and the network core.
-The nRF5340 Audio applications support the following types of DFU flash memory layouts:
-
-* Internal flash memory layout - which supports only single-image DFU.
-* External flash memory layout - which supports :ref:`multi-image DFU <ug_nrf5340_multi_image_dfu>`.
-
-
-Enabling FOTA upgrades
-**********************
+The nRF5340 Audio application supports two strategies for updating the applications on the application and network core.
+Which of the two strategies are used is controlled by setting the kconfig AUDIO_DFU to the appropriate setting.
 
-The FOTA upgrades are only available when :ref:`nrf53_audio_app_building_script`.
-With the appropriate parameters provided, the :file:`buildprog.py` Python script will add overlay files for the given DFU type.
-To enable the desired FOTA functions:
+* Single-image upgrade - Uses the internal flash to upgrade either the application core or network core separately. Selected by setting :kconfig:option:`CONFIG_AUDIO_DFU` to 1.
```

Comment:
Should we mention that this is dangerous in case the hci_rpmsg version changes?

Reply:
Updated warning message to sound a bit more like an encouragement for the user to investigate potential risks